gnovm/pkg/gnolang/machine.go

```diff
@@ -402,9 +402,15 @@
 	// Files' package names must match the machine's active one.
 	// if there is one.
 	for _, fn := range fns {
-		if fn.PkgName != "" && fn.PkgName != m.Package.PkgName {
-			panic(fmt.Sprintf("expected package name [%s] but got [%s]",
-				m.Package.PkgName, fn.PkgName))
+		var (
+			notEmptyName = fn.PkgName != ""
```

Comment:
Am I crazy or is this line (and the one after it) misaligned?
Can you run `make fmt`?

Reply:
I'm pretty sure this is the reason the linter is failing